packages/studio/frontend/src/components/SuperInput/EditorFrame.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { forwardRef } from 'react'
-import * as style from './EditorFrame.module.scss'
+import style from './EditorFrame.module.scss'
```

Comment:
For the style here to be properly recognized and loaded by parcel, you must `import * style from ...`. See this doc for reference: [https://parceljs.org/languages/sass/ ](https://parceljs.org/languages/sass/ )and the part where they mention CSS modules.